checkstyle/import-control.xml

```diff
@@ -181,13 +181,9 @@
 
   <subpackage name="metadata">
     <allow pkg="org.apache.kafka.clients" />
-    <allow pkg="org.apache.kafka.common.feature" />
```

Comment:
Why are we removing import controls?

Reply:
the reason for removing this is that I noticed that the metadata module doesn't actually import anything from `org.apache.kafka.common.feature` and so forth, so there is no reason to have it here.